conda_rpms/generate.py

```diff
@@ -49,17 +49,19 @@ def render_dist_spec(dist, config):
                                 rpm_prefix=rpm_prefix,
                                 install_prefix=install_prefix)
 
-def render_env(branch_name, label, repo, config, tag, commit_num):
+
+def render_env(branch_name, label, config, tag, commit_num):
+    install_prefix = config['install']['prefix']
+    rpm_prefix = config['rpm']['prefix']
     env_info = {'url': 'http://link/to/gh',
                 'name': branch_name,
-		'label' : label,
-                'summary': 'A SciTools environment.',
+                'label' : label,
+                'summary': 'A {} environment.'.format(rpm_prefix),
                 'version': commit_num,}
-    install_prefix = config['install']['prefix']
-    rpm_prefix = config['rpm']['prefix']
+    tag_name, = tag.split('-', 2)[2:]
```

Comment:
please may you add a `#` comment line explaining why this split and index operation is coded in this way, perhaps with an example, as in the test case below
`env-default-2016_12_15-2`


Alternatively, this could be a regular expression, explicitly splitting out
{env}, {env_name}, {datestamp and optional indexer}
with an exception handler if this re.match failed